src/main.rs

```diff
@@ -19,7 +19,9 @@ use jujutsu::ui::Ui;
 fn run(ui: &mut Ui) -> Result<(), CommandError> {
     let app = default_app();
     let (command_helper, matches) = parse_args(ui, app, std::env::args_os())?;
-    run_command(ui, &command_helper, &matches)
+    let result = run_command(ui, &command_helper, &matches);
+    ui.finalize_writes()?;
```

Comment:
nit: The example applications have the call in main() instead. I suppose the advantage of having it in `run()` is that we also handle errors from `finalize_writes()`. So do the same in the example applications?

Reply:
Hm, actually, this is probably better in `main()`, because we have to make sure that `finalize_writes()` is always called, even if `create_ui()` or `parse_args()` fails.